lib/tether/tether.go

```diff
@@ -323,6 +326,25 @@ func (t *tether) initializeSessions() error {
 				session.wait = &sync.WaitGroup{}
 				session.extraconfigKey = name
 
+				if session.Diagnostics.SysLogConfig != nil {
+					cfg := session.Diagnostics.SysLogConfig
+					var w syslog.Writer
+					if writer == nil {
+						writer, err = syslog.Dial(cfg.Network, cfg.RAddr, syslog.Info|syslog.Daemon, fmt.Sprintf("%s", t.config.ID[:shortLen]))
+						if err != nil {
+							log.Warnf("could not connect to syslog server: %s", err)
+						}
+						w = writer
+					} else {
+						w = writer.WithTag(fmt.Sprintf("%s", t.config.ID[:shortLen]))
+					}
+
+					if w != nil {
+						stdout.Add(w)
+						stderr.Add(w)
```

Comment:
syslog is not enabled for exec sessions. I only enabled them for non-exec sessions: https://github.com/vmware/vic/pull/5222/files#diff-1e478a53415085b3136db3d38acbba32R42